doc/source/whatsnew/v0.20.3.txt

```diff
@@ -59,7 +59,7 @@ I/O
 
 Plotting
 ^^^^^^^^
-
+- Allow series plots to accept RGB and RGBA tuples as color arguments (:issue:`16233`)
```

Comment:
I would that the sentence with "Fix bug" or "Fix regression in ..", as it now sounds that this a new thing that it is allowed.